light-client/src/config/utils.rs

```diff
@@ -20,7 +25,13 @@ where
 {
     let bytes_opt: Option<String> = serde::Deserialize::deserialize(deserializer)?;
     if let Some(bytes) = bytes_opt {
-        Ok(Some(hex_str_to_bytes(&bytes).unwrap()))
+        match hex_str_to_bytes(&bytes) {
+            Ok(b) => Ok(Some(b)),
```

Comment:
We prefer not to use single-character variable names in this codebase whenever possible, if you could update this from `b` to `value` that would be better